_delphi_utils_python/delphi_utils/validator/static.py

```diff
@@ -230,7 +230,7 @@ def find_all_unexpected_geo_ids(df_to_test, geo_regex, geo_type):
                         ValidationFailure(
                             "check_geo_id_type",
                             filename=nameformat,
-                            message="geo_ids saved as floats; strings preferred"))
+                            message=f"{len(leftover)} geo_ids saved as floats; strings preferred"))
```

Comment:
keep the old message prefix, and take out the `len()` to just list the offending ids
```suggestion
                            message=f"geo_ids saved as floats; strings preferred: {leftover}"))
```